src/TLabel.cpp

```diff
@@ -114,3 +146,60 @@ void TLabel::enterEvent(QEvent* event)
     }
     QWidget::enterEvent(event);
 }
+
+bool TLabel::forwardEventToMapper(QEvent* event)
+{
+    if (event->type() == QEvent::MouseButtonPress || event->type() == QEvent::MouseButtonRelease || event->type() == QEvent::MouseMove)
```

Comment:
🤔 Looks like another case where `switch` specifically `switch(event->type())` might be slightly cleaner...!

Reply:
I think this is OK... we're running the same code on all of them.